diff --git a/node_modules/spdx-correct/index.js b/node_modules/spdx-correct/index.js
index c51a79f..5effafd 100644
--- a/node_modules/spdx-correct/index.js
+++ b/node_modules/spdx-correct/index.js
@@ -186,6 +186,10 @@ var transforms = [
         .replace(/ (\d)/, '-$1')
         .replace(/ ?International/, '') +
       '-4.0'
+  },
+  // e.g. 'GNU (LESSER GENERAL PUBLIC LICENSE | LGPL)
+  function (argument) {
+    return argument.replace(/GNU (lesser general public license|lgpl).*/gi, 'LGPL-3.0-or-later')
   }
 ]
 
